examples/AzureSDKDemoSwift/Source/Common.swift

```diff
@@ -96,7 +96,16 @@ struct AppState {
                 tenant: AppConstants.tenant, clientId: AppConstants.clientId, application: application,
                 account: AppState.currentAccount
             )
-            let options = StorageBlobClientOptions(logger: ClientLoggers.none)
+            let downloadPolicy = TransferNetworkPolicy(transferOver: [.wifiOrEthernet], enableAutoResume: [])
```

Comment:
Since these are both parameters of the same type, maybe we should give them similar name. E.g. `transferOn` and `autoResumeOn` ?

Reply:
Done